Packs/Cyberint/ReleaseNotes/1_0_3.md

```diff
@@ -0,0 +1,56 @@
+
+#### Integrations
+##### Cyberint
+- Updated the mapper, classifier, layout and added new incidents fields.
+
+#### Classifiers
+##### Cyberint (mapper)
+- Maintenance and stability enhancements.
+##### Cyberint (classifier)
+- Updated classifier according to new schema.
+
+#### Layouts
+##### Cyberint Incident Layout
+ - Updated incident and indicator layouts to content pack format.
+<!--
+##### layoutscontainer-Cyberint_Incident_Layout.json
+ - Updated incident and indicator layouts to content pack format.
+-->
+
+#### Incident Types
+##### Cyberint Incident
+- Updated incident and indicator layouts to content pack format.
+
+#### Incident Fields
+##### incident_createddate
+- Added **Created Date** to Incident Fields
```

Comment:
I'm not sure we should mention new entities in the RN. Does it fail without it?

Reply:
Yes... 

Validating Packs/Cyberint/ReleaseNotes/1_0_3.md as releasenotes
id set validations for: Packs/Cyberint/ReleaseNotes/1_0_3.md
Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Target vector" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Cyberint Category" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Vulnerable CName Record" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "IOC" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Target Brand" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Source Category" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Impacts" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Related Entities" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "CyberintConfidence" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Recommandation" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Alert_Attachments" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Ticket_ID" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Expert Analysis" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "Created Date" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the indicatorfield "Created By" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the mapper "Cyberint (mapper)" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the layoutscontainer "Cyberint Incident Layout" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items

Packs/Cyberint/ReleaseNotes/1_0_3.md: [RN107] - No release note entry was found for the incidentfield "ClosureReason" in the Cyberint pack. Please rerun the update-release-notes command without -u to generate an updated template. If you are trying to exclude an item from the release notes, please refer to the documentation found here - https://xsoar.pan.dev/docs/integrations/changelog#excluding-items